### PR TITLE
Conclude Coffee Hispanic Tournament

### PR DESCRIPTION
**Pull request by fajar13k**
- Move directory structures
- CHT cleanup
- Write CHS article
- Add CHS to Tournaments index

### CODE REVIEW

File: wiki/Tournaments/CH/CHT_1/en.md [hunk 1]
```diff
@@ -393,9 +394,9 @@ Monday, 27 January 2020:
 | #22 | ![][flag_PE] [Leongf](https://osu.ppy.sh/users/9103105) | 192 |
 | #23 | ![][flag_PE] [Waynedude](https://osu.ppy.sh/users/9138491) | 211 |
 
-- Only the top 8 from each tiers can proceed to form a team of 2 and move on to Knockout Stage
-- More detailed qualifiers result can be accessed via this [spreadsheet](https://docs.google.com/spreadsheets/d/1wGCrW2IdRmmnu_JKuAVEELSKvCm2zacyiLDLns0SwsM/edit#gid=1549366682)
-- Average is determined by `Σ position earned when playing the qualifiers mappool (in each tiers)`, the lower the score the better
+- Only the top 8 from each tiers could proceed to form a team of 2 and move on to knockout stage
+- More detailed qualifiers result could be accessed via this [spreadsheet](https://docs.google.com/spreadsheets/d/1wGCrW2IdRmmnu_JKuAVEELSKvCm2zacyiLDLns0SwsM/edit#gid=1549366682)
```
Comment (TPGPL) [inline wiki/Tournaments/CH/CHT_1/en.md:398]:
```suggestion
- More detailed qualifiers result can be accessed via this [spreadsheet](https://docs.google.com/spreadsheets/d/1wGCrW2IdRmmnu_JKuAVEELSKvCm2zacyiLDLns0SwsM/edit#gid=1549366682)
```

File: wiki/Tournaments/CH/CHS_1/en.md [hunk 2]
```diff
@@ -0,0 +1,524 @@
...
+  - **[Camellia - LET'S JUMP (Camellia's "diffractive" Hardcore Remix) (Moko) \[Isomorphic Substitution\]](https://osu.ppy.sh/beatmapsets/1052797#osu/2326935)**
+  - **[GET IN THE RING - Midnight Syndrome (Atalanta) \[Desire\]](https://osu.ppy.sh/beatmapsets/862929#osu/1886627)**
+
+### Finals
+
+**[Download the mappack here! (156 MB)](https://drive.google.com/file/d/1g847SBMY7dh1rO1PD4cGzIWu_8ymbunN/view)**
+
+- NoMod
+  - [Twin-Tails - Twintail Dreamer! (Minorsonek) \[diuS' Expert\]](https://osu.ppy.sh/beatmapsets/773026#osu/1680262)
+  - [GLORYHAMMER - Rise of the Chaos Wizards](https://osu.ppy.sh/beatmapsets/990188#osu/2164735)
```
Comment (TPGPL) [inline wiki/Tournaments/CH/CHS_1/en.md:133]: diff is missing

**Review by TPGPL — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/CH/CHS_1/en.md (hunk 2 above)
Comment (fajar13k) [reply wiki/Tournaments/CH/CHS_1/en.md:133]: Which the mapper probably deleted the map after the map's been used, what's the best action here?

Comment (TPGPL) [reply wiki/Tournaments/CH/CHS_1/en.md:133]: if it can't be accessed then its fine as it is i suppose

**Review by TPGPL — APPROVED** [review]

**Pull request merged**